test/generators/suspenders/factory_bot_generator_test.rb

```diff
@@ -0,0 +1,215 @@
+require "test_helper"
+require "generators/suspenders/factory_bot_generator"
+
+module Suspenders
+  module Generators
+    class FactoryBotGenerator::DefaultTest < Rails::Generators::TestCase
+      include Suspenders::TestHelpers
+
+      tests Suspenders::Generators::FactoryBotGenerator
+      destination Rails.root
+      setup :prepare_destination
+      teardown :restore_destination
```

Comment:
`destination` is set to `test/dummy`, which is a blank Rails app.
`setup` and `teardown` handle adding or removing files needed for this specific test to run. For example, adding the `Gemfile`.